### PR TITLE
Updated emission lines and continuum models

### PR DESCRIPTION
**Pull request by HiramHerrera**
The following Pull Request updates:

-  `BossDr9_fiducial_continuum` model dispersions based on the results found by Rodrigo in slide 6 of [this presentation](https://desi.lbl.gov/trac/attachment/wiki/LyafMeetings/min21-03-23/Realism_QSO_template_2021_03_23.pdf) to reproduce eBOSS DR16 QSO dispersion found by PCA method.

- Emission line model, in order to better reproduce the results of PICCA Mean Flux on EDR data. 

| LyA Region       | LyB region       |
| -------------- | -------------- |
| image  | image |

This new model also addresses the correlation between CIV Equivalent Width and Magnitude as found on observational data. Previous mocks didn't have such correlation
image

### CODE REVIEW

File: py/desisim/scripts/lya_simqso_model.py [hunk 1]
```diff
@@ -127,16 +127,20 @@ def EmLineTemplate_modified_develop(*args,**kwargs):
...
+                                'NIII991':1.8,#2.2,
+                                #'unknownI':1.2,
+                                #'LyB/OIVn':1.2,'LyB/OIVb':1.2,
+                                'NII':0.8,
+                                'FeIII:UV1n':0.1,
+                                'FeIII:UV1b':0.1,
+                                'LyAb':1.2,'LyAn':1.2,
+                                'CIII*':1.7
+                                })
+    #kwargs['EmissionLineTrendFilename']=resource_filename('desisim', 'data/emlinetrends_Harris2016mod')
```
Comment (alxogm) [inline py/desisim/scripts/lya_simqso_model.py:141]: Please remove these commented lines, its not necessary to keep them

File: py/desisim/scripts/lya_simqso_model.py [hunk 2]
```diff
@@ -127,16 +127,20 @@ def EmLineTemplate_modified_develop(*args,**kwargs):
...
+    kwargs['EmissionLineTrendFilename']=resource_filename('desisim', 'data/emlinetrends_EDRv2')
```
Comment (alxogm) [inline py/desisim/scripts/lya_simqso_model.py:142]: Maybe  we should call the file emlinetrends_Harris2016mod_v2, as this new file is a modified version of the previous one.

**Review by alxogm — CHANGES_REQUESTED** [review]

**Comment by andreufont** [comment]
Happy to merge once Alma is happy, she know more than I do about this part of the code.

I wouldn't worry too much about the "mean continuum" plot above, since the data has redshift errors that are not present in the mocks (are they?), and as we saw in Youles et al. (2022) this has an effect in the mean continuum measured by Picca.

**Review by alxogm — APPROVED** [review]

**Comment by alxogm** [comment]
I think you can merge @HiramHerrera

**Pull request merged**